src/contractTest/java/uk/gov/hmcts/futurehearings/hmi/cdc/consumer/HearingAPIConsumerTest.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.futurehearings.hmi.cdc.consumer;
+
+import static uk.gov.hmcts.futurehearings.hmi.cdc.consumer.common.TestingUtils.readFileContents;
+
+import uk.gov.hmcts.futurehearings.hmi.Application;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.Pact;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.model.RequestResponsePact;
+import net.serenitybdd.rest.SerenityRest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ActiveProfiles("contract")
+@SpringBootTest(classes = {Application.class})
+@ExtendWith(PactConsumerTestExt.class)
+@ExtendWith(SpringExtension.class)
+public class HearingAPIConsumerTest {
+
+    @Value("${targetHost}")
+    private String targetHost;
+
+    @Value("${targetSubscriptionKey}")
+    private String targetSubscriptionKey;
+
+    private static final String PROVIDER_REQUEST_HEARING_API = "/hmi/hearings";
+
+    private Map<String, String> headersAsMap = new HashMap<>();
+
+    @BeforeEach
+    public void initialiseValues() {
+        headersAsMap.put("Host", targetHost);
+        headersAsMap.put("Ocp-Apim-Subscription-Key", targetSubscriptionKey);
+        headersAsMap.put("Ocp-Apim-Trace", "true");
+        headersAsMap.put("Company-Name", "HMCTS");
+        headersAsMap.put("Content-Type", "application/json");
+        headersAsMap.put("Source", "SnL");
+        headersAsMap.put("Destination", "CFT");
+        headersAsMap.put("DateTime", "datetimestring");
+        headersAsMap.put("RequestType", "TypeOfCase");
+
+    }
+
+   @Pact(provider = "SandL_API", consumer = "HMI_HEARING_API")
```

Comment:
Consumer should be HMI - Why is HEARING_API appended, Restrict it to the System
Producer should be SandL as there is not conceptual or logical system name as SandLAPI...